src/main/java/sx/blah/discord/api/internal/DiscordUtils.java

```diff
@@ -701,11 +705,17 @@ private static IPresence getPresenceFromJSON(GameObject game, String status) {
 	 */
 	public static IEmoji getEmojiFromJSON(IGuild guild, EmojiObject json) {
 		long id = Long.parseUnsignedLong(json.id);
-		List<IRole> roles = Arrays.stream(json.roles)
+		IRole[] roles = Arrays.stream(json.roles)
 				.map(role -> guild.getRoleByID(Long.parseUnsignedLong(role)))
-				.collect(Collectors.toList());
+				.toArray(IRole[]::new);
```

Comment:
What's the point of changing this?

Reply:
Almost everything related to roles for Emojis is stored as an array and it is easier to convert an array to a list than the other way around. This could easily be changed back, I would just need to convert it.

Reply:
Almost everything related to roles for Emojis is stored as an array and it is easier to convert an array to a list than the other way around. This could easily be changed back, I would just need to convert it.